blog/templates/blog/blog_page.html

```diff
@@ -1,9 +1,27 @@
-{% extends "super.html" %}
+{% extends "base.html" %}
 {% load wagtailcore_tags %}
 
-{% block body %}
-    <h1>{{ page.title }}</h1>
-    {% for block in page.body %}
-        <section>{% include_block block %}</section>
-    {% endfor %}
+{% block main %}
+	<h1 class="article-title">{{ page.title }}</h1>
+
+	<article class="article-content">
+		<!-- Details table-->
+		{% include "blog/_details_table.html" with blog=page only %}
+
+
+		{% if page.introduction %}
+				<p class="heading">{{ page.introduction }}</p>
+		{% endif %}
+
+		<section class="article-content__inner">
+			{% include 'common/_video_or_image.html' with video=page.primary_video image=page.teaser_image caption=page.image_caption %}
```

Comment:
I am not sure this `video_or_image` component matches the design exactly, the font size is (I think) one step too large and it does not include the image attribution.